packages/wagmi/exports/vue.ts

```diff
@@ -4,27 +4,18 @@ import type {
   W3mConnectButton,
   W3mNetworkButton
 } from '@web3modal/scaffold'
-import { onUnmounted, reactive, ref } from 'vue'
 import type { Web3ModalOptions } from '../src/client.js'
 import { Web3Modal } from '../src/client.js'
-import { VERSION } from '../src/utils/constants.js'
+import { VERSION } from '@web3modal/utils'
 
 // -- Types -------------------------------------------------------------------
 export type { Web3ModalOptions } from '../src/client.js'
 
-type OpenOptions = Parameters<Web3Modal['open']>[0]
-
-type ThemeModeOptions = Parameters<Web3Modal['setThemeMode']>[0]
-
-type ThemeVariablesOptions = Parameters<Web3Modal['setThemeVariables']>[0]
-
-declare module '@vue/runtime-core' {
-  export interface ComponentCustomProperties {
-    W3mConnectButton: Pick<W3mConnectButton, 'size' | 'label' | 'loadingLabel'>
-    W3mAccountButton: Pick<W3mAccountButton, 'disabled' | 'balance'>
-    W3mButton: Pick<W3mButton, 'size' | 'label' | 'loadingLabel' | 'disabled' | 'balance'>
-    W3mNetworkButton: Pick<W3mNetworkButton, 'disabled'>
-  }
+export interface ComponentCustomProperties {
```

Comment:
Same comment about these and @vue/runtime-core, as well as moving them to vue scaffold